postgres_service/utils.py

```diff
@@ -0,0 +1,26 @@
+def parse_endpoint(endpoint):
```

Comment:
You probably want to use a URL/URI parsing library for this. Hyperlink is good and part of Twisted: http://hyperlink.readthedocs.io/en/latest/